### ISSUE TITLE
pyenv woes

### ISSUE THREAD

**Issue by neverfox** [neovim/neovim#8253]
I originally posted my issue to SpaceVim but realize this may be more of a neovim issue. See here for a complete rundown:

https://github.com/SpaceVim/SpaceVim/issues/1621

**Comment by justinmk** [comment]
DId you search the issue tracker for existing (closed) discussions? Try 0.2.3 and `:checkhealth`.

If you still have a problem, please open a new issue which includes the answers to the issue template.

**Comment by neverfox** [comment]
Yes, I did and found quite a few. However, none of them seemed to come to any resolution that works currently. If you know of one in particular, I would appreciate knowing where to find it. As for `:checkhealth`, that was mentioned in the issue and in particular why that's part of the problem. I will try 0.2.3, as that's probably the one thing I haven't tried (I'll have to do a HEAD installation as homebrew, which I use, is only up to 0.2.2). Also, sorry I didn't fill out the template as I thought the fact that I had filled out Spacevim's first might have provided you with most of the same information you're after. I'll fill out yours if I need to reopen.
